packages/avatars/src/styled/StyledAvatar.ts

```diff
@@ -250,19 +252,19 @@ export const StyledAvatar = styled.figure.attrs({
   }
 
   &::before,
-  & > img {
+  && > img {
     border-radius: inherit;
     width: 100%;
     height: 100%;
   }
 
-  & > img {
+  && > img {
     box-sizing: inherit;
     vertical-align: bottom;
     object-fit: cover;
   }
 
-  & > svg {
+  && > svg {
```

Comment:
Can these double `&&` hacks be avoided by applying `!important` to the width/height? In this case I think that would be preferable, and more bullet-proof styling.

Reply:
Since this is applied to nested elements, I don't think we would have to worry about these specificity issues being beaten. With `<figure />` being the wrapping element I think we should limit `!important` overrides to that element for now.

Reply:
The problem is that we've learned that double/triple specificity hacks are a real pain for customization. In this case, I suspect the hack is being applied to items where it is not needed (`box-sizing`, `vertical-align`, `object-fit`?). It turns out `!important offers a cleaner path for customization overrides, should those ever be attempted.